yarn.lock

```diff
@@ -3315,6 +3340,21 @@ chokidar@^2.0.2, chokidar@^2.0.4:
   optionalDependencies:
     fsevents "^1.2.7"
 
+chokidar@endiliey/chokidar#master:
```

Comment:
writing it here so I won’t forget. This should be #2.x

Need to update yarn.lock